tests/unit/test_services.py

```diff
@@ -964,6 +964,71 @@ def test_fix_doesnt_fail_on_empty_file() -> None:
     assert result == source
 
 
+def test_fix_remove_unused_imports() -> None:
```

Comment:
If this is the default behaviour then it's already covered in the rest of the tests. Thus I'll remove this one